test-app/yarn.lock

```diff
@@ -1416,7 +1416,7 @@ function-bind@^1.1.1:
   resolved "https://registry.yarnpkg.com/function-bind/-/function-bind-1.1.1.tgz#a56899d3ea3c9bab874bb9773b7c5ede92f4895d"
   integrity sha512-yIovAzMX49sF8Yl58fSCWJ5svSLuaibPxXQJFLmBObTuCr0Mf1KiPopGM9NiFjiYBCbfaa2Fh6breQ6ANVTI0A==
 
-"functionless@file:..":
+functionless@../:
```

Comment:
hmm, why did this change?

Reply:
The other path didn't install for me, but I think they are the same?

Reply:
Renamed.